UefiCpuPkg/Library/MpInitLib/MpLib.h

```diff
@@ -997,24 +997,19 @@ RemoveNxprotection (
   IN UINTN                 Length
   );
 
-// MU_CHANGE START: Enable removal of NX attribute from buffer
-
 /**
-  Remove NX attribute from Buffer and apply RO to Buffer
+Add ReadOnly protection to the range specified by BaseAddress and Length.
 
-  @param[in]  Buffer      Buffer whose attributes will be altered
-  @param[in]  Size        Size of the buffer
+The PEI implementation uses CpuPageTableLib to change the attribute.
+The DXE implementation uses gDS to change the attribute.
 
-  @retval EFI_SUCCESS             NX attribute removed, RO attribute applied
-  @retval EFI_INVALID_PARAMETER   Buffer is not page-aligned or Buffer is 0 or Size of buffer
-                                  is not page-aligned
-  @retval Other                   Return value of LocateProtocol, ClearMemoryAttributes, or SetMemoryAttributes
+@param[in] BaseAddress  BaseAddress of the range.
+@param[in] Length       Length of the range.
 **/
-EFI_STATUS
-BufferRemoveNoExecuteSetReadOnly (
-  IN EFI_PHYSICAL_ADDRESS  Buffer,
-  IN UINTN                 Size
+VOID
+ApplyReadOnlyMemoryProtection (

```

Comment:
```suggestion
VOID
EFIAPI
ApplyReadOnlyMemoryProtection (
```

Reply:
Since this is consumed from inside of the PciMpLib and DxeMpLib, do we need the EFIAPI? What I mean is that its not a external interface to the library.

Reply:
No you don't. I saw MpLib.h and thought this was a public header at the time.